### ISSUE TITLE
Long text overflow on `tctl` output

### ISSUE THREAD

**Issue by klizhentas** [gravitational/teleport#4677]
Long roles should cap the output to match the terminal width, instead they overlfow on the next line:

```bash
$ tctl get roles --format=text
Role  Allowed to login as      Node Labels Access to resources                                                                                                                                                                  
----- ------------------------ ----------- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
admin {{internal.logins}},root <all nodes> role:list,create,read,update,delete,auth_connector:list,create,read,update,delete,session:list,read,trusted_cluster:list,create,read,update,delete,event:list,read,token:create,read 
```

**Comment by zmb3** [comment]
@lxea I think you may have recently fixed this - can you confirm?

**Comment by lxea** [comment]
I've added a fix for this in #9519  now. It also adds a verbosity flag to `tctl get` but it only does anything with `tctl get roles`.

**Comment by zmb3** [comment]
This has been fixed, long lines are now truncated.
